docs/docusaurus/versioned_docs/version-1.7.0/feg/s1ap_federated_test.md

```diff
@@ -174,14 +171,28 @@ make
 exit
 ```
 
-- Traffic vm:
+- Traffic VM:
 
 ```bash
 cd magma/lte/gateway
 vagrant up magma_trfserver
+```
 
-# exit from vagrant vm
+#### Run test manually
+
+Once you have built all of the VMs, you can try to run a test from the
+`magma_test` VM:
+
+```bash
+cd magma/lte/gateway
+vagrant ssh magma_test
+
+# inside vagrant vm
+cd magma/lte/gateway/python/integ_tests
+make integ_test TESTS=federated_tests/s1aptests/test_attach_detach.py
+
+# once the test is done, you can exit the vagrant vm
 exit
 ```
 
-You can then [Run test manually](#Run-test-manually)
+This will run a simple attach-detach test.
```

Comment:
Could be integrated in lines 183 & 184:
`... run the simple attach-detach test from the ... manually.`

Reply:
Thanks for the suggestion. I've just made the change.